app/Commands/Sites/CreateCommand.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace App\Commands\Sites;
+
+use App\Commands\BaseCommand;
+use App\Commands\Concerns\HasServerIdParameter;
+use App\Helpers\OptionsHelper;
+use App\Questions\Ask;
+use App\Questions\Choice;
+use App\Questions\Confirm;
+use App\Questions\HasQuestions;
+use Illuminate\Support\Str;
+
+class CreateCommand extends BaseCommand
+{
+    use HasQuestions;
+    use HasServerIdParameter;
+
+    protected $signature = 'sites:create
+                            {installation_method : Type of installation (wp or blank)}
+                            {server_id? : Server ID}
+                            {--domain= : Domain name}
+                            {--site_user=}
+                            {--db_name=}
+                            {--db_user=}
+                            {--db_pass=}
+                            {--wp_title=}
+                            {--wp_admin_user=}
+                            {--wp_admin_email=}
+                            {--wp_admin_pass=}
+                            {--php_version=}
+                            {--page_cache_enabled}
+                            {--https_enabled}
+                            {--profile=}
```

Comment:
Adding descriptions for these 12 parameters like we have for `--domain` would be helpful.

Reply:
I'm wondering if `wp_title` might make more sense as `site_title`